Add: new meeting notes (#161)
datproject/comm-comm: meeting-notes/56-02Jul2020.md

    - local-first browser
    - would like to use extension-messages. Being able to re-use peers connections.
    - aiming for having lower level access than what beaker does. So apps can re-use that is working on node now.
    - would like to partner with other projects, help share other projects to share stuff on the web.
    - also would like to collaborate more with `gateway-browser` around fetch api improvements.

## metanomial's project

- metanomial: peer api has been a bit unstable
- mauve: `hyper-presence` module has been helpful. Using flooding.
- metanomial: what's the max latency?
- mauve: if 100ms is acceptable to you, you would be fine. 0 control of the graph of connections. Would like to do some tests. It hasn't been a problem yet.
- latency could be a bit worse than peersockets. If you have less than 100 peers, latency likely wont be a problem.
- MMST lib: tries to reduce overall number of connections while keeping network relatively dense. It would be nice to get that inside hyperswarm but that is a lot of work.
- metanomial shares: `hyper://609a82a6d3df2b564754b0c96d080024cf8e03e042b32a7b0b7c5539b4ae46f7/`

## standards, beaker

> - mauve: people feel a bit minimal/powerless compared with bit standards like w3c.

Reviewer: ```suggestion
- mauve: people feel a bit minimal/powerless compared with big standards bodies like w3c.
```

- mauve: standards base approach is slow. Longer term this would be standardized

> - mauve: one way to tackle the standards questions, has an approach more collaborative, eg: would like to work with gateway-browser, or have a way to standardize http api/usage with other projects like sonar so these projects can work together. collaborative with more angles at once could be more helpful. work together with other protocols.

Reviewer: ```suggestion
- mauve: one way to tackle the standards questions, is a more collaborative approach, eg: would like to work with gateway-browser, or have a way to standardize http api/usage with other projects like sonar so these projects can work together. collaborative with more angles at once could be more helpful. work together with other protocols.
```
